functions/routes/index.js

```diff
@@ -7,10 +7,28 @@ const {
     login, signUp, image, getAll,
 } = require('../controllers/user/index');
 
+const {
+    newTopic, getTopics,
+} = require('../controllers/topic/index');
+
+const {
+    newPost, getPosts,
+} = require('../controllers/post/index');
+
+const {
+    newComment, getComments,
+} = require('../controllers/comment/index');
+
 // user Routes
 router.post('/signup', signUp);
 router.post('/login', login);
 router.put('/user/image', authGuard, image);
 router.get('/users', getAll);
+router.post('/new_topic', newTopic);
+router.get('/get_topics', getTopics);
+router.post('/new_post', newPost);
+router.get('/get_posts', getPosts);
+router.post('/new_comment', newComment);
+router.get('/get_comments/:postID', getComments);
```

Comment:
is it possible to modularize this like have a 
routes/user.js containing all the routes for user
routes/comment.js containing routes for comment
routes/post.js containing routes for post

all you have in routes/index.js is to call the module for user, comment and post